lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore

```diff
@@ -25,7 +25,7 @@
                             <%- gettext('Cohort Name') %> *
                             <span class="sr"><%- gettext('(Required Field)')%></span>
                     </label>
-                    <input name="cohort-name" value="<%= cohort_name_value %>" class="input cohort-name" id="cohort-name"
+                    <input name="cohort-name" value="<%- cohort_name_value %>" class="input cohort-name" id="cohort-name"
```

Comment:
@robrap I think this is correct, does it look right to you? Screenshots form running this code locally:

"View Source" does not have the dynamic content:
<img width="1033" alt="screen shot 2016-03-21 at 2 23 09 pm" src="https://cloud.githubusercontent.com/assets/7373924/13929508/dcdb6ea0-ef70-11e5-8440-22880f39607c.png">

"Inspect" seems to show non-escaped value:
<img width="1397" alt="screen shot 2016-03-21 at 2 23 33 pm" src="https://cloud.githubusercontent.com/assets/7373924/13929530/f0b9e78a-ef70-11e5-8e36-122aadda3d5c.png">